src/Tools/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/AnalyzerTest`1.cs

```diff
@@ -163,8 +163,19 @@ protected async Task VerifyDiagnosticsAsync((string filename, SourceText content
                         .Select(x => IsInSourceFile(x, sources) ? x.WithLineOffset(1) : x)
                         .ToArray();
 
-                    var commentPrefix = Language == LanguageNames.CSharp ? "//" : "'";
-                    VerifyDiagnosticResults(await GetSortedDiagnosticsAsync(sources.Select(x => (x.filename, x.content.Replace(new TextSpan(0, 0), $" {commentPrefix} <auto-generated>\r\n"))).ToArray(), additionalFiles, additionalMetadataReferences, analyzers, cancellationToken).ConfigureAwait(false), analyzers, expectedResults);
+                    if (Exclusions.HasFlag(AnalysisExclusions.GeneratedCode))
```

Comment:
If this flag *isn't* set, is it worthwhile to verify that the generated code in fact *does* produce the warnings?